Sources/OpenAPIURLSession/URLSessionTransport.swift

```diff
@@ -193,7 +193,18 @@ extension URLRequest {
         }
         self.init(url: url)
         self.httpMethod = request.method.rawValue
-        for header in request.headerFields { setValue(header.value, forHTTPHeaderField: header.name.canonicalName) }
```

Comment:
Would it also get fixed if we used the `addValue` API instead? https://developer.apple.com/documentation/foundation/urlrequest/2011522-addvalue

Reply:
It seems that the `addValue` API might be a better option. ChatGPT suggested something related to `Cookie` and `Set-Cookie` headers, but I didn’t find any relevant information in the official documentation. I’m not sure if there might be other issues involved here.

Reply:
addValue use `,` but cookie needs `; ` 💔

Reply:
@guoye-zhang can you help here?

Reply:
I opened #51 and borrowed the code to do the correct transaction from swift-http-types